Tests/SentryTests/Integrations/Performance/Network/SentryNetworkTrackerIntegrationTests.swift

```diff
@@ -173,10 +173,10 @@ class SentryNetworkTrackerIntegrationTests: XCTestCase {
         dataTask.resume()
         wait(for: [expect], timeout: 5)
         
-        let children = Dynamic(transaction).children as [Span]?
-        
-        XCTAssertEqual(children?.count, 1) //Span was created in task resume swizzle.
-        let networkSpan = try XCTUnwrap(children?.first)
+        let children = try XCTUnwrap(Dynamic(transaction).children as [Span]?)
```

Comment:
Could this also be an optional cast?

```suggestion
        let children = try XCTUnwrap(Dynamic(transaction).children as? [Span])
```

Reply:
Nope, that doesn't compile

```
Cast from 'Dynamic' to unrelated type '[any Span]' always fails
```